.github/workflows/on-push.yml

```diff
@@ -75,8 +75,41 @@ jobs:
 
       - name: Unit test
         run: |
-          yarn test
+          yarn test:unit
 
+  visual-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout ${{ github.ref }}
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: ${{ env.NODE_VERSION_FILE }}
+          cache: ${{ env.PACKAGE_MANGER }}
+          cache-dependency-path: ${{ env.PACKAGE_MANGER_LOCK_FILE }}
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile --immutable
+
+      - name: Install Playwright Browsers
+        run: yarn playwright install --with-deps
```

Comment:
Do we need to install playwright locally also?
If so, then I must ask you to add some information about visual tests in README
- that you need to install playwright
- that those tests need to be prepared for each page - and special cases on that page
- that if the tests will fail on CI you can check them by running some command and update snapshot by running some other command

Reply:
Yes. But i think we can accomplished this by using 
```
    "postinstall": "npx playwright install --with-deps"
```

So it will be install after `yarn install` so no need to installing it manually on your own and no need to install it as separate step in Github Actions. 

Changes pushed.

Reply:
Changes pushed